templates/locations/icemap.html

```diff
@@ -49,13 +49,19 @@
          <div class="btn-group pull-right">
             <span class="hidden-xs">
                <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                  {{ "Undanfarna viku" if period == 'week' else "Síðasta sólarhring" }} 
+                  {% if period == 'week' %}
+                     Undanfarna viku
+                  {% elif period == 'month' %}
+                     Síðastliðinn mánuð
```

Comment:
Strictly speaking it's the last 30 days, no?

Reply:
Indeed :)